src/main/kotlin/lotto/domain/Prize.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.domain
+
+enum class Prize(val money: Int, val matchCount: Int) {
+    FIRST(2000000000, 6),
```

Comment:
```suggestion
    FIRST(2_000_000_000, 6),
```
언더스코어를 활용하면 가독성을 살릴 수 있습니다!